src/applications/login/components/MockAuthButton.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import environment from 'platform/utilities/environment';
+import ENVIRONMENT_CONFIGURATIONS from 'site/constants/environments-configs';
+
+export default function MockAuthButton() {
+  return (
+    <button
+      type="button"
+      aria-label="Mock Authentication"
+      className="usa-button mauth-button vads-u-margin-y--1p5 vads-u-padding-y--2"
+      onClick={() => {
+        const environ =
+          typeof process !== 'undefined'
+            ? process.env.BUILDTYPE
+            : environment.BUILDTYPE;
+        window.location.replace(
```

Comment:
We need to keep the window History for the session (ie when a person hits the back button they land where they're supposed to).  You can change this to `window.location = <url>`